app/src/main/java/org/oppia/app/player/content/ContentListFragmentPresenter.kt

```diff
@@ -0,0 +1,62 @@
+package org.oppia.app.player.content
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import org.oppia.app.application.ApplicationContext
+import org.oppia.app.databinding.ContentListFragmentBinding
+import org.oppia.data.backends.gae.model.GaeSubtitledHtml
+import javax.inject.Inject
+
+/** Presenter for [ContentListFragment]. */
+class ContentListFragmentPresenter @Inject constructor(
+  @ApplicationContext private val context: Context,
+  private val fragment: Fragment
+) {
+
+  private val entity_type: String = "exploration"
+  private val entity_id: String = "umPkwp0L1M0-"
+
+  var contentList: MutableList<GaeSubtitledHtml> = ArrayList()
+
+  fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
+    val binding = ContentListFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
+    binding.recyclerView.apply {
+      binding.recyclerView.layoutManager = LinearLayoutManager(context)
+      binding.contentCardAdapter = ContentCardAdapter(context, entity_type, entity_id, contentList);
+    }
+    fetchDummyExplorations()
+    return binding.root
+  }
+
+  // TODO (#121) :Replace this once interface for ExplorationDataController is available
+  private fun fetchDummyExplorations() {
```

Comment:
Can we fetch this from ``ExplorationDataController``, instead? Let's go ahead and change those explorations to contain any additional data we need to properly test this code (feel free to copy the html samples you have below into those explorations if needed).

Reply:
Having problem in fetching explorations. Its returning null.

Reply:
now fetching data from exploration